fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/Function.java

```diff
@@ -46,4 +58,16 @@ public boolean isHighOrder() {
     public final String getName() {
         return name;
     }
+
+    public Expression withConstantArgs(Literal literal) {
+        return this;
```

Comment:
return this is not a good idea. i think we should throw exceptions if on function not support Monotonicity